pages/categories/downstream.md

```diff
@@ -10,12 +10,97 @@ Downstream emissions (Category D) are the emissions produced by using an organis
+| ------------------ | ------------------ | ------------------ |
+| Desktop            | 72 - 100           | 72.3               |
+| Laptop             | 15 - 30            | 17.1               |
+| Tablet             | 3 - 5.5            | 3                  |
+| Smartphone         | 0.77 - 2           | 1                  |
+| Monitor            | 17 - 30            | 30                 |
+| Television         | 40 - 120           | 74                 |
+
+Based on Scope3 and Urban et al's [^urban] installed base data:
+
+| Device             | Installed Base     |
+| ------------------ | ------------------ |
+| Desktop            | 37%                |
+| Laptop             | 63%                |
+| Monitor            | 52%                |
+
+When the end user’s computer cannot be determined between a laptop (with or without a monitor) and a desktop, we can use the installed base data to estimate that a personal computer consumes an average of 53.2W (rounded up):
+
+$$
+(0.37 \times 72.3) + (0.63 \times 17.1) + (0.52 \times 30) = 53.12W
+$$
+
+With these values, it is possible to estimate the power consumption by users when using a product or service on a device. The more information available about what devices are used and how much time users spend on these products, the more accurate the estimates can be.
+
+To calculate energy use (Wh), the power (W) of the device is divided by the time factor (hours = 1, minutes = 60, seconds = 3600), then multiplied by the duration. For example, assume a blog page with an estimated reading time of 5 minutes. The energy consumption of a user on a laptop (without a monitor) can be estimated using the following calculation:
+
+$$ Energy\ (Wh) = 17.1W / 60 \times 5 = 1.4Wh $$
+
+Examples of device energy consumption per minute:
+
+| Device             | Power (W)          | Time               | Energy (Wh)        |
+| ------------------ | ------------------ | ------------------ | ------------------ |
+| Personal Computer  | 53.2               | 1 minute           | 0.89               |
+| Desktop + Monitor  | 102.3              | 1 minute           | 1.71               |
+| Laptop + Monitor   | 47.1               | 1 minute           | 0.79               |
+| Laptop             | 17.1               | 1 minute           | 0.29               |
+| Tablet             | 3                  | 1 minute           | 0.05               |
+| Smartphone         | 1                  | 1 minute           | 0.02               |
+
+
+{% include linkedHeading.html heading="End-User Device Carbon Emissions" level=3 %}
+Using the user's location, the [carbon intensity](/glossary#carbon-intensity) of the energy grid can be used to calculate the carbon emissions of the device.
+
+Continuing the example above, a user, in the UK reading a blog page on a laptop for 5 minutes; we can calculate the estimated carbon emissions based on the energy consumed (1.4Wh = 0.0014kWh) and the UK energy grid (238gCO2e/kWh):
```

Comment:
No comma after "user" _or_ another comma after "UK"
I'd suggest a colon rather than a semicolon, or a full stop and a new sentence.